packages/@aws-cdk/aws-sns/lib/subscription.ts

```diff
@@ -36,7 +36,21 @@ export interface SubscriptionOptions {
    *
    * @default - all messages are delivered
    */
-  readonly filterPolicy?: { [attribute: string]: SubscriptionFilter };
+  readonly filterPolicy? : { [attribute: string]: SubscriptionFilter };
+
+  /**
+   * The filter policy. V2 is compatible with FilterPolicyScope set to "MessageBody"
+   *
+   * @default - all messages are delivered
```

Comment:
Adding a `@deprecated` to the jsdoc causes errors in the unit tests as any use of `filterPolicy` throws an error. I believe that is the correct practice but would this not break any existing uses of `filterPolicy`?